### ISSUE TITLE
RFC: Add endpoint to get all flair ranks

### ISSUE THREAD

**Issue by TimJentzsch** [GrafeasGroup/blossom#168]
Add an endpoint to retrieve all flair ranks like Green and Diamond. We need this in a lot of places so I think it would make sense to expose them in the API to have them consistent everywhere. It should list the name of all ranks together with their gamma thresholds.

We'll need colors too eventually, but it probably makes more sense to add them client-side because they can depend on the color theme that is used.

**Comment by itsthejoker** [comment]
What would be an example use case for pulling this information from the API instead of embedding them into the client service? Being able to update them once and have it update everywhere?

**Comment by TimJentzsch** [comment]
Yes, otherwise we would have to update it in Blossom, in Buttercup, potentially in Bubbles and then in any third party tools. I guess it doesn't update that often, but still it would be nice to have IMO.
